test/cql-pytest/test_aggregate.py

```diff
@@ -217,3 +222,22 @@ def test_avg_decimal_2(cql, table1, cassandra_bug):
 def test_reject_aggregates_in_where_clause(cql, table1):
     assert_invalid_message(cql, table1, 'Aggregation',
                            f'SELECT * FROM {table1} WHERE p = sum((int)4)')
+
+# Reproduces #13265
+# Aggregates must handle case-sensitive column names correctly.
+def test_sum_case_sensitive_column(cql, table2):
+    p = unique_key_int()
+    cql.execute(f'insert into {table2} (p, c, somecolumn, "SomeColumn", "OtherColumn") VALUES ({p}, 1, 1, 10, 100)')
```

Comment:
I'd opt for extracting the query string into a separate var and then parameterizing it, something like:
```
query = f'insert into {table2} (p, c, somecolumn, "SomeColumn", "OtherColumn") VALUES ({p}, %s, %s, %s, %s)'
cql.execute(query % (1, 1, 10, 100))
cql.execute(query % (2, 2, 20, 200))
etc.
```
because if someone needs to change the string, he'll need to do it in 3 places, not just 1.

Reply:
I'm also not sure if `c` column is all that needed (perhaps I missing its purpose), but if you want to keep it, why not assigning it other values than those for `p` col?

Reply:
The philosophy of cql-pytest is to try to share test tables as much as possible, because this is what allows individual test functions to take less than 0.1 seconds (creating and deleting a table is one of the slowest things most tests do). Here "table2" isn't shared by anything else, but hopefully in the future it will be shared by additional tests. And "c" is what helps share it - each of the tests pick a unique "p" and then this one can have three items with different c in the same p.

Regarding parametrizing the test - the best tests are, in my opinion, short and easy to read. Nobody every wants to change the test - but people will need to read it in the future. I'm fine with the duplication of the INSERT line three times, I think it's one line shorter than your suggestion and easier to read, so I personally prefered @cvybhu approach. But I think your suggestion is ok too if @cvybhu likes it.

Reply:
I also think that the existing code is more readable, I like to have everything in one place, it's easier on the brain.
The duplication isn't that bad, and in case of an error, the error message will contain the whole query, which is nice to have.

Reply:
The `c` is there so that I can have 3 rows in the same partition. Inserting 3 rows with the same `p` would result with a single row, as there can be only one row with the same primary key.